app/models/breed.rb

```diff
@@ -4,6 +4,11 @@ class Breed < ActiveRecord::Base
   has_many :cat_breeds
   has_many :cats, through: :cat_breeds
 
+  validates :name, length: { minimum: 2, maximum: 32 }
+  validates_uniqueness_of :name
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.